### PR TITLE
Added README , SUPPORT, SECURITY, CODE_OF_CONDUCT, CONTRIBUTING 

### PR DESCRIPTION
**Pull request by Saumya-Sunder**
Added README , SUPPORT, SECURITY, CODE_OF_CONDUCT, CONTRIBUTING 

Signed-off-by: Saumya Sunder <saumya.sunder@seagate.com>

### CODE REVIEW

File: SUPPORT.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (sachinpunadikar) [inline SUPPORT.md:1]: Is it left blank intentionally ?

**Review by sachinpunadikar — COMMENTED** [review]
found that support.md blank

File: SUPPORT.md (hunk 1 above)
Comment (Saumya-Sunder) [reply SUPPORT.md:2]: Added it. I think I missed it out while copying.

**Comment by Saumya-Sunder** [comment]
Done @sachinpunadikar, please check.

**Review by sachinpunadikar — APPROVED** [review]
+1
Look at the codacy issues - https://app.codacy.com/gh/Seagate/cortx-dsal/pullRequest?prid=6285547

**Review by s-arya — APPROVED** [review]

**Pull request merged**